CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - New "attestation" section in node JSON configuration to specify remote endpoint required to retrieve the endorsement certificates for SEV-SNP attestation report (#4277).
+- The `ccf::RpcContext` now contains functionality for storing user data with `set_user_data` and retrieving it with `get_user_data` (#4291).
```

Comment:
I'm tempted to suggest renaming to `set/get_ephemeral_user_data()` to make it abundantly clear that the persistence of this data is in no way guaranteed. But I may be worrying too much.

Reply:
I'd think being on the rpc context would indicate that it won't stay around afterwards. I'm not too sure on which other name it should have either though. Is there a way to mark it experimental so that the naming could change later without being breaking?

Reply:
I've just looked at the set_* calls on the RpcContext, and I think you're absolutely right.